uportal-war/src/main/java/org/apereo/portal/shell/PortalShell.java

```diff
@@ -76,9 +76,6 @@ public static void main(String[] args) throws Exception {
                         new CompilerConfiguration(System.getProperties());
                 final GroovyShell shell = new GroovyShell(binding, conf);
                 shell.run(scriptFile, args);
-            } else {
```

Comment:
Is it necessary to remove this part?

I'm not positive, but it looks as though it may be connected to an Ant task that launches an open-ended groovy shell with uPortal in the classpath.  (I'm pretty sure a task like that exists, or once did.)